src/main/resources/templates/bo/pages/promotion.html

```
@@ -2,46 +2,383 @@
 <!-- Promotion Page Content Fragment -->
 <div th:fragment="content" id="page-promotion" class="page-section hidden flex-1 overflow-y-auto p-8 space-y-6">
-    <div class="grid grid-cols-3 gap-6">
-        <div class="bg-indigo-900 text-white p-6 rounded-xl shadow-md">
-            <div class="text-indigo-200 text-xs font-bold uppercase mb-1">지급 대기 포인트</div>
-            <div class="text-3xl font-bold">125,000 P</div>
-            <div class="mt-4 text-sm text-indigo-200">자동 지급 실패: <span class="text-white font-bold">2건</span></div>
+
+    <!-- Sub-tab Navigation -->
+    <div class="border-b border-slate-200">
+        <nav class="flex gap-1" id="promotion-tabs">
+            <button onclick="switchPromotionTab('discount')"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)